actors/eam/src/lib.rs

```diff
@@ -0,0 +1,376 @@
+use std::iter;
+
+use fil_actors_evm_shared::address::EthAddress;
+use num_traits::Zero;
+
+use ext::{
+    account::PUBKEY_ADDRESS_METHOD,
+    evm::RESURRECT_METHOD,
+    init::{Exec4Params, Exec4Return},
+};
+use fil_actors_runtime::{
+    actor_dispatch_unrestricted, deserialize_block, extract_send_result, AsActorError,
+};
+
+use fvm_ipld_encoding::ipld_block::IpldBlock;
+use fvm_shared::{error::ExitCode, sys::SendFlags};
+use serde::{Deserialize, Serialize};
+
+pub mod ext;
+
+use {
+    fil_actors_runtime::{
+        actor_error,
+        runtime::builtins::Type,
+        runtime::{ActorCode, Runtime},
+        ActorError, EAM_ACTOR_ID, INIT_ACTOR_ADDR, SYSTEM_ACTOR_ADDR,
+    },
+    fvm_ipld_encoding::{strict_bytes, tuple::*, RawBytes},
+    fvm_shared::{
+        address::{Address, Payload},
+        crypto::hash::SupportedHashes,
+        ActorID, METHOD_CONSTRUCTOR,
+    },
+    num_derive::FromPrimitive,
+};
```

Comment:
Nit: this import format with the `use` block is novel here. Is there a tool that can rewrite the imports canonically for us?

Reply:
I'm happy to fix this manually.